utils/data.py

```
@@ -0,0 +1,25 @@
+DATA_DIR = "data"
+os.makedirs(DATA_DIR, exist_ok=True)
+
+def get_file_path(filename: str) -> str:
+    return os.path.join(DATA_DIR, filename)
+
+def load_data(filename: str) -> List[Dict[str, Any]]:
+    path = get_file_path(filename)
+    if not os.path.exists(path):
+        return []
+    try:
+        with open(path, "r", encoding="utf-8") as f:
```

Ruff (0.14.13)

Unnecessary mode argument

Remove mode argument

(UP015)